pkg/kapp/resources/resource_filter.go

```diff
@@ -92,6 +94,26 @@ func (f ResourceFilter) Matches(resource Resource) bool {
 		}
 	}
 
+	if len(f.Labels) > 0 {
+		var matched bool
```

Comment:
Should we introduce a separate method for this flag like BoolFilter? Maybe we can do this later and do it for a few other flags as well.

Reply:
However, the code is readable, so we can definitely skip it.